doc/adrs/113_use_dynamic_handles_to_connect_edges_between_nodes.adoc

```diff
@@ -0,0 +1,134 @@
+= ADR-113 - Use dynamic handles to connect edges between nodes.
```

Comment:
We're at 117 now, this one should be number 118.
No "." at the end of the title.
It is not referenced in the CHANGELOG for 2023.12.0.